bindings/node/src/mongocrypt.cc

```diff
@@ -418,15 +419,46 @@ MongoCrypt::MongoCrypt(const CallbackInfo& info)
         }
     }
 
+    if (options.Has("csfleSearchPaths")) {
+        Napi::Value search_paths_v = options["csfleSearchPaths"];
+        if (!search_paths_v.IsArray()) {
+            throw TypeError::New(Env(), "Option `csfleSearchPaths` must be an array");
+        }
+        Array search_paths = search_paths_v.As<Array>();
+        for (uint32_t i = 0; i < search_paths.Length(); i++) {
+            mongocrypt_setopt_append_csfle_search_path(
+                _mongo_crypt.get(),
+                search_paths.Get(i).ToString().Utf8Value().c_str());
+        }
```

Comment:
@kevinAlbs Can you confirm that it is expected that `mongocrypt_setopt_append_csfle_search_path()` is exposed in this way (or a similar one)? I want to make sure because it’s not listed in the scope, but is part of the mongocrypt.h public API.

Reply:
Yes, there is no plan to change the declaration or behavior of [mongocrypt_setopt_append_csfle_search_path](https://github.com/mongodb/libmongocrypt/blob/f1582c5a6fc09bbdda3f84382d4eed55f78af93e/src/mongocrypt.h.in#L440-L443).